test/no-functions.js

```diff
@@ -5,13 +5,17 @@ var Understudy = require('../').Understudy;
 var actor = new Understudy();
 
 actor.before('no-functions', function () {
-  assert.equal(true, false);
+  throw new Error('before shouldn\'t be reached');
 });
 
-actor.after('no-functions', function () {
```

Comment:
I believe you also want to make the same change you made to the `actor.before` statement above.